apis/python/src/tiledbsoma/tiledb_array.py

```diff
@@ -177,3 +177,27 @@ def _chunk_is_contained_in(
                 return False
 
         return True
+
+    def _consolidate_and_vacuum_fragment_metadata(self) -> None:
+        """
+        This post-ingestion helper consolidates and vacuums fragment metadata -- this is quick to
+        do, and positively impacts query performance.  It does _not_ consolidate bulk array data,
+        which is more time-consuming and should be done at the user's opt-in discretion.
+        """
+
+        if self.uri.startswith("tiledb://"):
```

Comment:
Go ahead and remove this. No need to differentiate here.

Reply:
@Shelnutt2 

```
>>> soma.obs._consolidate_and_vacuum_fragment_metadata()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/johnkerl/git/single-cell-data/TileDB-SOMA/apis/python/src/tiledbsoma/tiledb_array.py", line 195, in _consolidate_and_vacuum_fragment_metadata
    tiledb.consolidate(self.uri, ctx=ctx)
  File "tiledb/libtiledb.pyx", line 3860, in tiledb.libtiledb.consolidate
  File "tiledb/libtiledb.pyx", line 3947, in tiledb.libtiledb._consolidate_timestamp
  File "tiledb/libtiledb.pyx", line 451, in tiledb.libtiledb._raise_ctx_err
  File "tiledb/libtiledb.pyx", line 436, in tiledb.libtiledb._raise_tiledb_error
tiledb.cc.TileDBError: C API: TileDB Internal, std::exception; Consolidation is not supported for remote arrays.
```

Reply:
@johnkerl, Seth indicated that support for `tiledb://` consolidating/vacuuming is complete and will be rolled out soon, so we can drop this exception.